common/scripts/push-csv.sh

```diff
@@ -16,20 +16,20 @@
 #
 
 set -e
-QUAY_NAMESPACE=${QUAY_NAMESPACE:-opencloudio}
```

Comment:
Similar to above, this file likely isn't used anymore so it doesn't matter what you do with it, but the change here to ICR doesn't really make sense (this also interacts with the old deprecated application packages on quay)